e2e/helpers.py

```diff
@@ -0,0 +1,118 @@
+import boto3
+import os
+import random
+import cis_crypto
+from cis_profile import common
+from cis_profile import profile
+
+
+def get_cis_environment():
+    return os.getenv("CIS_ENVIRONMENT", "development")
+
+
+def get_client_id_path():
+    return "/iam/cis/{}/change_client_id".format(get_cis_environment())
+
+
+def get_client_secret_path():
+    return "/iam/cis/{}/change_service_client_secret".format(get_cis_environment())
+
+
+def get_url_dict():
+    cis_environment = get_cis_environment()
+
+    if cis_environment == "development":
```

Comment:
```suggestion
    # XXX Use well-known endpoint library for fetching URLs and audience
    if cis_environment == "development":
```
adding that here in case you don't actually use the wk lib, so that its clear one shouldnt copy these urls ;-)